app/js/smaaResults/smaaResults.html

```diff
@@ -12,4 +12,5 @@
   scales="workspace.scales.base"
   current-subproblem="subProblem"
   subproblems="subProblems"
+  can-edit="editMode.canEdit"
```

Comment:
Input the editmode, in closed it has more properties. I think putting in a simplification now will bite us in the ass later.